fe/fe-core/src/main/java/org/apache/doris/common/util/DebugUtil.java

```diff
@@ -27,7 +27,7 @@
 import java.util.UUID;
 
 public class DebugUtil {
-    public static final DecimalFormat DECIMAL_FORMAT_SCALE_3 = new DecimalFormat("#.000");
+    public static final DecimalFormat DECIMAL_FORMAT_SCALE_3 = new DecimalFormat("0.000");
```

Comment:
Why change this? Is it some thing wrong with origin `#.000`?

Reply:
It will display incorrectly when the value is less than 1. For example, if the value is 0.12, it will display **.120** instead of **0.120** .